Tests/FluentAssertions.Specs/Numeric/NumericDifferenceAssertionsSpecs.cs

```diff
@@ -0,0 +1,703 @@
+            }
+
+            [Fact]
+            public void When_a_nullable_ulong_difference_between_small_numbers_it_should_not_add_difference_message()
+            {
+                // Arrange
+                ulong? value = 7;
+                const ulong expected = 4;
+
+                // Act
+                Action act = () =>
+                    value.Should().Be(expected, "because we want to test the failure {0}", "message");
+
+                // Assert
+                act
+                    .Should().Throw<XunitException>()
+                    .WithMessage($"Expected value to be {expected}UL because we want to test the failure message, but found {value}UL.");
+            }
+
+            [Fact]
+            public void When_a_nullable_ulong_difference_between_large_numbers_it_should_add_difference_message()
+            {
+                // Arrange
+                ulong? value = 50;
+                const ulong expected = 20;
+
+                // Act
+                Action act = () =>
+                    value.Should().Be(expected, "because we want to test the failure {0}", "message");
+
+                // Assert
+                act
+                    .Should().Throw<XunitException>()
+                    .WithMessage($"Expected value to be {expected}UL because we want to test the failure message, but found {value}UL (difference of {value - expected}).");
+            }
+
+            [Fact]
+            public void When_a_nullable_ushort_difference_between_large_numbers_it_should_add_difference_message()
+            {
+                // Arrange
+                ushort? value = 11;
+                const ushort expected = 2;
+
+                // Act
+                Action act = () =>
+                    value.Should().Be(expected, "because we want to test the failure {0}", "message");
+
+                // Assert
+                act
+                    .Should().Throw<XunitException>()
+                    .WithMessage($"Expected value to be {expected}us because we want to test the failure message, but found {value}us (difference of {value - expected}).");
+            }
+
+            [Fact]
+            public void When_a_double_difference_between_large_numbers_it_should_add_difference_message()
+            {
+                // Arrange
+                const double value = 1.5;
+                const double expected = 1;
+
+                // Act
+                Action act = () =>
+                    value.Should().Be(expected, "because we want to test the failure {0}", "message");
+
+                // Assert
+                act
+                    .Should().Throw<XunitException>()
+                    .WithMessage($"Expected value to be 1.0 because we want to test the failure message, but found {value} (difference of {value - expected}).");
```

Comment:
Formatting floating points is culture sensitive due to different decimal point separators.
So this would fail with e.g. Danish culture as `Be` would use the english `.` as decimal separator, but `{value}` would be formatting as `1,5` in Danish.

The solution is to remove the string interpolations and embed the expected values directly.
E.g. 
```cs
WithMessage("Expected value to be 1.0 because we want to test the failure message, but found 1.5 (difference of 0.5).")
```